### PR TITLE
Add presentation material to session detail

### PR DESCRIPTION
**Pull request by MasayukiSuda**
## Issue
- close https://github.com/DroidKaigi/conference-app-2020/issues/164

## Overview (Required)
- Display a disabled link even if there is no material yet.
- Display a link if there is material.

## Screenshot
Before | no material | no material dark | has material | has material dark 
:--: | :--: | :--: | :--: | :--: 
 |  |  |  |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: corecomponent/androidcomponent/src/main/res/navigation/navigation.xml [hunk 1]
```diff
@@ -102,6 +102,16 @@
             app:popEnterAnim="@anim/fade_in"
             app:popExitAnim="@anim/fade_out"
             />
+        <action
+            android:id="@+id/action_session_to_chrome"
```
Comment (takahirom) [inline corecomponent/androidcomponent/src/main/res/navigation/navigation.xml]: 👍

**Comment by takahirom** [comment]
Sorry, I tried resolving conflict with GitHub. But probably I broke it 🙇 
Can you reset and fix the conflict? (You can use force push 🙇 )

**Comment by MasayukiSuda** [comment]
@takahirom 
Thanks code review and resolving conflict.
I tried resolving conflict. Please check again.🙇

**Comment by takahirom** [comment]
LGTM! 🆒

**Pull request merged**